src/seasons/Season.java

```diff
@@ -0,0 +1,35 @@
+package seasons;
+
+public enum Season {
+    SPRING("wiosna"),
+    SUMMER("lato"),
+    AUTUMN("jesień"),
+    WINTER("zima");
+
+    private String description;
```

Comment:
brakuje miesiecy

Reply:
Czy miesiaca maja byc dodane do opisow enumow, czy stworzyc oddzielna tablice z miesiacami i porami roku skad beda wywalywane?

Reply:
to może być pole ktore jest tablica napisów

Reply:
poprawione